watched.js

```diff
@@ -31,9 +31,12 @@ const parseDom = (dom) => {
         for (movie of moviesNodes) {
             let title = movie.querySelector("div.preview__originalTitle")?.textContent;
             if (!title) {
-                title = movie.querySelector("h2.preview__title").textContent;
+                title = movie.querySelector("div.preview__alternateTitle")?.textContent;
```

Comment:
Jeśli zmienili tylko nazwę klasy, to wystarczy tylko w 32 linijce zamienić `originalTitle` na `alternateTitle`, dodawanie nowego `if` jest chyba zbędne